src/containers/Profile/components/NetworkStats/NetworkStats.module.scss

```diff
@@ -28,7 +28,7 @@
 .networkStatCircle {
   display: flex;
   flex-direction: column;
-  padding: 0 $base-spacing;
+  padding: 0 $s-spacing;
   align-items: center;
```

Comment:
i did this to keep the external link icon on the same line as the text -- otherwise it wrapped it to a new line